### PR TITLE
Updated and added Umbraco.Sys

### PR DESCRIPTION
**Pull request by rasmuseeg**
Please fill in this template.

- [ ] Prefer to make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code.
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [ ] Run `npm run lint -- package-name` if a `tslint.json` is present.

If adding a new definition:
- [ ] The package does not provide its own types, and you can not add them.
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Run `tsc` without errors.
- [ ] Include the required [files](https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package) and header. Base these on the README, *not* on an existing project.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.

### CODE REVIEW

File: umbraco/umbraco-resources.d.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
-// Type definitions for Umbraco v7.2.8
+// Type definitions for Umbraco v7.5.0
 // Project: https://github.com/umbraco
 // Definitions by: DeCareSystemsIreland <https://github.com/DeCareSystemsIreland>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Definitions: https://our.umbraco.org/apidocs/ui/#/api/umbraco.resources
```
Comment (andy-ms) [inline umbraco/umbraco-resources.d.ts]: Don't need this line.

File: umbraco/umbraco.d.ts [hunk 2]
```diff
@@ -0,0 +1,116 @@
+// Type definitions for Umbraco v7.5.0
```
Comment (andy-ms) [inline umbraco/umbraco.d.ts]: This should be added to `index.d.ts` instead. Having `foo/foo.d.ts` was the old style, but now we use `index.d.ts` instead.

**Review by ghost — COMMENTED** [review]
Would be nice to add `tslint.json` too.

**Comment by mhegazy** [comment]
Thanks for your contribution. This PR requires some changes and has not had any activity for the last 3 weeks. For housekeeping purposes we are closing stale PRs. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

**Pull request closed without merging**